libs/ui/src/global_styles.tsx

```diff
@@ -152,10 +152,10 @@ export const GlobalStyles = createGlobalStyle<GlobalStylesProps>`
     --focus-outline: ${(p) =>
       p.isTouchscreen
         ? `${p.theme.colors.primary} dashed ${p.theme.sizes.bordersRem.medium}rem`
-        : 'none'};
+        : `${p.theme.colors.primary} solid ${p.theme.sizes.bordersRem.medium}rem`}
   }
 
-  :focus {
+  :focus-visible {
```

Comment:
This change helped make SearchSelect focus correctly. Details on why this is useful here: https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible#focus_vs_focus-visible